Revise resource descriptions and add new resources (#1829)
elixir-europe/rdmkit: pages/national_resources/no_resources.md

      Your_tasks: [data_analysis]
    url: https://documentation.sigma2.no/hpc_machines/hardware_overview.html
  - name: Norwegian Research and Education Cloud (NREC)

>     description: NREC (formerly known as UH-IaaS) is a collaboration project between the University of Bergen and the University of Oslo, with additional contributions from NeIC (Nordic e-Infrastructure Collaboration) and Uninett. We've been in production since 2016 and are currently providing cloud infrastructure for several high profile academic projects, including CERN's ALICE and ATLAS experiments. Our hardware is located exclusively on-premise, our services are developed locally and we are almost entirely based on Open Source Software and open standards, making us a more transparent alternative to commercial cloud providers.

Reviewer: ```suggestion
    description: NREC (formerly known as UH-IaaS) is a collaboration project between the University of Bergen and the University of Oslo, with additional contributions from NeIC (Nordic e-Infrastructure Collaboration) and Uninett. It has been in production since 2016 and is currently providing cloud infrastructure for several high-profile academic projects, including CERN's ALICE and ATLAS experiments. The hardware is located exclusively on-premise, our services are developed locally, and the infrastructure is almost entirely based on open-source software and open standards, making this a more transparent alternative to commercial cloud providers.
```

Reviewer: Unclear what is meant by "on premise" and "locally", as this is a collaborative effort.

Author: We'll technically there a two "on premise"...
